_dashboards/drag-drop-wizard.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Drag and drop wizard (experimental feature)
+nav_order: 15
+---
+
+# Build visualizations by  dragging and dropping fields
+
+You can use the drag and drop visualization Wizard in OpenSearch Dashboards to create your data visualizations easily with just a few clicks or a single mouse gesture. 
+
+The drag and drop visualization Wizard:
+
+* Gives you immediate an view of your data without the need to preselect the visualization output 
+* Gives you the flexibility to change visualization types and index patterns on the fly
+* Gives you the ability to easily navigate between multiple screens 
+
+Beginning with OpenSearch release 1.3, the drag and drop wizard is available as an experimental feature. We invite you to try the wizard and let us know how you feel about it.
```

Comment:
add this note syntax directly on next line below sentence to create a callout:
{: .note}